scalding-core/src/test/scala/com/twitter/scalding/StringUtilityTest.scala

```diff
@@ -0,0 +1,95 @@
+package com.twitter.scalding
+
+import org.scalatest.{ Matchers, WordSpec }
+
+class StringUtilityTest extends WordSpec with Matchers {
```

Comment:
we don't want to support regexs for splits. So it should match as long is the separator is not a regex.